code/game/objects/items/stacks/medical.dm

```diff
@@ -93,7 +93,7 @@
 			patient.balloon_alert(user, "not hurt!")
 			return FALSE
 		user.visible_message("<span class='infoplain'><span class='green'>[user] applies [src] on [patient].</span></span>", "<span class='infoplain'><span class='green'>You apply [src] on [patient].</span></span>")
-		patient.heal_bodypart_damage((heal_brute * 0.5))
```

Comment:
Does this not double the healing power of the suture, calculated for a normal human?

A human maxHealth is 100, so your new calculation there is heal_brute * 1

Reply:
![image](https://github.com/tgstation/tgstation/assets/34697715/6e734578-1f49-4cfa-b6a2-649ff52921f7)

why are you calculating for a normal human if it only affects simple animals and basic mobs

Reply:
because I'm not paying attention